### PR TITLE
Improved ItemStages docs.

### PR DESCRIPTION
**Pull request by Darkhax**
This PR adds some info about previously undocumented ZenMethods. It also adds some info about an issue people frequently bug me with.

### CODE REVIEW

File: docs/Mods/GameStages/ItemStages/ItemStages.md [hunk 1]
```diff
@@ -1,8 +1,34 @@
...
+mods.ItemStages.addItemStage("one", <minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 5 as short}]}));
+
+//Restricts access to all dyes in the ore dictionary.
+mods.ItemStages.addItemStage("one", <ore:dye>);
+```
+
+## Changing the name of item while hidden
+```
+//mods.ItemStages.setUnfamiliarName(newName, IIngredient);
+mods.ItemStages.setUnfamiliarName("Clump of Fur", <minecraft:wool:*>);
```
Comment (kindlich) [inline docs/Mods/GameStages/ItemStages/ItemStages.md]: You might want to add here whether that you can also set unlocalized strings
e.g. `mypack.itemstages.restricted.wool.restrictedName` and use lang files to localize?

File: docs/Mods/GameStages/ItemStages/ItemStages.md [hunk 2]
```diff
@@ -1,8 +1,34 @@
...
+
+## Staging a liquid
+```
+//mods.ItemStages.stageLiquid(stage, ILiquidStack);
+mods.ItemStages.stageLiquid("one", <liquid:water>);
+```
+
+## Staging a tooltip
+```
+//Note: Lines that start with the striction are hidden.
```
Comment (kindlich) [inline docs/Mods/GameStages/ItemStages/ItemStages.md]: Same as above, does this only affect localized strings or can you also compare against the unlocalised variants?

Comment (darkhax) [reply docs/Mods/GameStages/ItemStages/ItemStages.md]: Tooltip info doesn't have unlocalized keys it's just a list of arbitrary strings. So you would need to manually add lines to compare against new translations.

File: docs/Mods/GameStages/ItemStages/ItemStages.md (hunk 1 above)
Comment (darkhax) [reply docs/Mods/GameStages/ItemStages/ItemStages.md]: This also doesn't support translations at this time.

**Pull request merged**